gammapy/modeling/tests/test_sampler.py

```diff
@@ -19,11 +21,17 @@ def test_run(backend="ultranest"):
         )
         datasets.append(dataset)
 
-    datasets.models = [SkyModel.create(spectral_model="pl")]
-    datasets.models.parameters["index"].prior = UniformPrior(min=2, max=3)
-    datasets.models.parameters["amplitude"].prior = LogUniformPrior(
-        min=1e-12, max=1e-10
-    )
+    # test with linked parameters
```

Comment:
Ideally the linked parameter test should be differnt from the main test

Reply:
yes (I only modified the existing test because I was concerned about the time taken by the sampler but actually it is still reasonable)

Reply:
each tests takes only about 10sec so I added a 2nd one